lb-https.tf

```diff
@@ -6,6 +6,10 @@ variable "https_port" {
   default = "443"
 }
 
+variable "http_port" {
+  default = "80"
+}
+
```

Comment:
I think we should create a separate file for the HTTP specific config, and name it lb-http.tf.
Basically all the new configuration related to HTTP -> HTTPS redirect should go there.

Reply:
Done